recipe/meta.yaml

```diff
@@ -1,39 +1,34 @@
-{% set version = "1.4.7" %}
+{% set version = "1.4.8" %}
 
 package:
     name: kealib
     version: {{ version }}
 
 source:
     url: https://bitbucket.org/chchrsc/kealib/downloads/kealib-{{ version }}.tar.gz
-    sha256: ec38751b3b555d3a26f0c7445f2d2cd9d7c3a3502237519a206a50cb58df56ec
-    patches:
-        # create links libkea.so, libkea.so.1.4 etc. Fixed in next version
-        - soversion.patch
+    sha256: 0f24d8478865abcb17865c8f49c0370095726c529b8ac373ffae018ad3d40a02
 
 build:
-    number: 4
-    skip: True  # [win and py36]
-    features:
-        - vc9  # [win and py27]
-        - vc14  # [win and py35]
+    number: 0
+    run_exports:
+      # should match >=1.4.7,<1.5 ?
+      - {{ pin_subpackage('kealib', min_pin='1.4.7', max_pin='x.x') }}
```

Comment:
@gillins I guess we could use this opportunity to sync with the Anaconda recipe that already uses conda-build 3 format: https://github.com/AnacondaRecipes/kealib-feedstock/blob/master/recipe/meta.yaml

Ping @msarahan